app/controllers/sortable_tree_controller.rb

```diff
@@ -27,10 +26,12 @@ def sortable_tree(class_name, options = {})
           errors = []
           ActiveRecord::Base.transaction do
             records.each_with_index do |(record, parent_record), position|
-              record.send "#{options[:sorting_attribute]}=", position
-              if options[:tree]
-                record.send "#{options[:parent_method]}=", parent_record
+              if sorting_attribute
+                record.send "#{sorting_attribute}=", position
               end
+
+              record.send "#{parent_method}=", parent_record
```

Comment:
This is not backwards compatible, right? One could set option `tree` to `false` to only do sorting and not changing the nestedness, if I read the code correctly.

Reply:
`option[:tree]` was _always_ set to `true` regardless of the user value (ln 18). If the intention was to make updating the parent optional I suggest that it is (re)implemented by accepting `nil` for `:parent_method` just like for `:sorting_attribute`.

Reply:
Oops, you're right :)